translations/fr.json

```diff
@@ -0,0 +1,224 @@
+{
+    "translation": {
+        "🔵 Average": "🔵 Commun",
+        "🟣 Flea Banned": "🟣 Banni du marché aux puces",
+        "🟢 Great": "🟢 Peu Commun",
+        "⭐ Legendary ⭐": "⭐ Légendaire ⭐",
+        "🔴 Poor": "🔴 Mauvais",
+        "🛒 {{trader.name}} restock in {{numMinutes}} minutes 🛒": "🛒 {{trader.name}} se réapprovisionne dans {{numMinutes}} minute(s) 🛒",
+        "❌ ERROR ❌": "❌ ERREUR ❌",
+        "{{thingName}} set to {{level}}.": "IDK",
+        "A Dmg": "IDK",
+        "About": "À propos",
+        "additional results not shown.": "Les résultats additionels ne sont pas montrés.",
+        "All hideout stations set to {{level}}.": "IDK",
+        "All traders set to {{level}}.": "IDK",
+        "all traders": "tous les marchands",
+        "Ammo Table": "Tableau de munitions",
+        "An error has occurred": "Une erreur s'est produite",
+        "Available Commands": "Commandes disponibles",
+        "Barter": "Troquer",
+        "Bosses": "IDK",
```

Comment:
Bosses in this context means "challenging enemies". Think "bosses" as in a difficult boss to fight in a game